<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Нива</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="carConstEdit">
       <property name="placeholderText">
        <string>Constant attenuation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="carLinearEdit">
       <property name="placeholderText">
        <string>Linear attenuation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="carQuadEdit">
       <property name="placeholderText">
        <string>Quadratic attenuation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Настольная лампа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lampConstEdit">
       <property name="placeholderText">
        <string>Constant attenuation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lampLinearEdit">
       <property name="placeholderText">
        <string>Linear attenuation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lampQuadEdit">
       <property name="placeholderText">
        <string>Quadratic attenuation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Потолок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="roofConstEdit">
       <property name="placeholderText">
        <string>Constant attenuation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="roofLinearEdit">
       <property name="placeholderText">
        <string>Linear attenuation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="roofQuadEdit">
       <property name="placeholderText">
        <string>Quadratic attenuation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="carcasBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Каркасный режим</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="splitBox">
     <property name="text">
      <string>Нормальный светъ</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="carBox">
     <property name="text">
      <string>I drive</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
